### ISSUE TITLE
convert caffe model error

### ISSUE THREAD

**Issue by ghost** [xxworkspace/efficientnet_tf_pytorch_caffe#2]
hi @xxworkspace, when I try to run python pytorch2caffe_model.py efficientnet-b0, some errors show like following:
Check failed: registry.count(type) == 1 (0 vs. 1) Unknown layer type: Swish (known types: AbsVal, Accuracyat, ContrastiveLoss, Convolution, Crop, Data, Deconvolution, Dropout, DummyData, ELU, Eltwise, Embed, EuclideanLoss, Exp, Filter, Flatten, HDF5Data, HDF5OutnerProduct, Input, LRN, LSTM, LSTMUnit, Log, MVN, MemoryData, MultinomialLogisticLoss, PReLU, Parameter, Pooling, Power, RNN, ReLU, Reduction, Reshape, SPP,Slice, Softmax, SoftmaxWithLoss, Split, TanH, Threshold, Tile, WindowData)
*** Check failure stack trace: ***
Aborted (core dumped)
Do you know how to solve it? Thanks.

**Comment by ghost** [comment]
close
